src/__tests__/index.tsx

```diff
@@ -111,9 +115,9 @@ title: This is frontmatter
 
         <p>
           Here's a 
-          <strong>
+          <span>
             neat
-          </strong>
+          </span>
```

Comment:
Could the snapshot update automatically for you or did you have to manually update it (like I did)?

Reply:
I had to manually update it. Same error about stdin